rust/player/src/utils.rs

```diff
@@ -2172,5 +2166,53 @@ pub fn is_namespace_program_caller(ixns: &AccountInfo) -> bool {
         return false;
     };
 
-    return true;
+    true
+}
+
+/// Create account almost from scratch, lifted from
+/// https://github.com/solana-labs/solana-program-library/tree/master/associated-token-account/program/src/processor.rs#L51-L98
+#[inline(always)]
+pub fn create_or_allocate_account_raw<'a>(
```

Comment:
What is this for?

Reply:
Im not sure why its there, let me double check